docs/source/exportdatacode.rst

```diff
@@ -86,12 +88,13 @@ We are now going to use datalad to create a sibling dataset on OSF - this will c
 
     $ datalad create-sibling-osf OSF_PROJECT_NAME YOUR_OSF_REMOTE_NAME
```

Comment:
$datalad create-sibling-osf OSF_PROJECT_NAME -s YOUR_OSF_REMOTE_NAME ... otherwise defualts to 'osf'